src/sentry/api/endpoints/project_details.py

```diff
@@ -100,7 +100,42 @@ class ProjectMemberSerializer(serializers.Serializer):
     )
 
 
-@extend_schema_serializer(exclude_fields=["options"])
+@extend_schema_serializer(
+    exclude_fields=[
```

Comment:
As we keep iterating on the giant `Update a Project` API, this list should gradually decrease